mne/chpi.py

```diff
@@ -149,6 +150,115 @@ def head_pos_to_trans_rot_t(quats):
     return translation, rotation, t
 
 
+def _apply_quat(quat, pts, move=True):
+    """Apply MaxFilter-formatted head position parameters to points."""
+    trans = np.concatenate(
+        (quat_to_rot(quat[:3]),
+         quat[3:][:, np.newaxis]), axis=1)
+
+    return(apply_trans(trans, pts, move=move))
+
+
+def extract_head_pos_ctf(raw, gof_limit=0.98):
+    """Extract MaxFilter-formatted head position parameters from ctf dataset.
```

Comment:
We don't need to call them MaxFilter-formatted in the title. (You can do it in the notes if you want.) I think we're probably going to standardize around using the (N, 10) `ndarray` format. If we always labeled whose format it was (derived from) in the title, we'd have some long names...